index.src.html

```diff
@@ -174,79 +174,77 @@
-          DOMString                             attestationChallenge,
-          optional unsigned long                credentialTimeoutSeconds,
-          optional sequence < Credential >      blacklist,
-          optional FIDOExtensions               credentialExtensions
-      ); 
-   
-      Promise < FIDOAssertion > getAssertion ( 
+  partial interface Window {
+      readonly attribute WebAuthentication webauth;
+  };
+
+  interface WebAuthentication {
+      Promise < ScopedCredentialInfo > makeCredential (
+          Account                                 accountInformation,
+          sequence < ScopedCredentialParameters > cryptoParameters,
+          DOMString                               attestationChallenge,
+          optional unsigned long                  credentialTimeoutSeconds,
+          optional sequence < Credential >        blacklist,
+          optional WebAuthExtensions              credentialExtensions
+      );
+
+      Promise < WebAuthAssertion > getAssertion (
           DOMString                        assertionChallenge,
           optional unsigned long           assertionTimeoutSeconds,
-          optional sequence < Credential > whitelist, 
-          optional FIDOExtensions          assertionExtensions 
-      ); 
-  }; 
-   
-  interface FIDOCredentialInfo { 
-      readonly attribute Credential           credential; 
-      readonly attribute AlgorithmIdentifier  algorithm; 
-      readonly attribute any                  publicKey; 
-      readonly attribute AttestationStatement attestation; 
-  }; 
-
-  dictionary Account { 
-      required DOMString rpDisplayName; 
-      required DOMString displayName; 
-      DOMString          name; 
-      DOMString          id; 
-      DOMString          imageUri; 
+          optional sequence < Credential > whitelist,
+          optional WebAuthExtensions       assertionExtensions
+      );
   };
-   
-  dictionary FIDOCredentialParameters { 
-      required CredentialType        type; 
-      required AlgorithmIdentifier   algorithm; 
-  }; 
-
-  enum CredentialType { 
-      "FIDO" 
-  }; 
-
-  interface Credential { 
-      readonly attribute CredentialType type; 
-      readonly attribute DOMString      id; 
+
+  interface ScopedCredentialInfo {
+      readonly attribute Credential           credential;
+      readonly attribute AlgorithmIdentifier  algorithm;
+      readonly attribute any                  publicKey;
+      readonly attribute AttestationStatement attestation;
   };
-  
-  dictionary FIDOExtensions {
+
+  dictionary Account {
+      required DOMString rpDisplayName;
+      required DOMString displayName;
+      DOMString          name;
+      DOMString          id;
+      DOMString          imageUri;
+  };
+
+  dictionary ScopedCredentialParameters {
+      required CredentialType        type;
+      required AlgorithmIdentifier   algorithm;
+  };
+
+  enum CredentialType {
+      "ScopedUserCredential"
```

Comment:
For the sake of right now, going with @equalsJeffH 's `ScopedCred`, but I'm also a fan of `ScopedUser`.